web/src/app/shared/host-components-map/take.service.ts

```diff
@@ -165,10 +165,8 @@ export class TakeService {
           case 'depend':
             return depend();
           case 'odd':
-            return countRelations % 2 && countRelations >= a1
+            return (a1 === 0 || countRelations % 2) && countRelations >= a1
```

Comment:
this condition will be true for [0, odd] and all countRelations (for even numbers, too)